rust_src/src/buffers.rs

```diff
@@ -954,7 +955,7 @@ impl From<LispObject> for Option<LispOverlayRef> {
 impl LispMiscRef {
     pub fn as_overlay(self) -> Option<LispOverlayRef> {
         if self.get_type() == Lisp_Misc_Type::Lisp_Misc_Overlay {
-            unsafe { Some(mem::transmute(self)) }
+            Some(unsafe { mem::transmute(self) })
```

Comment:
Why is this one safe?

Reply:
It's not, I just re-added the `unsafe` inside the `Some` as a stylistic choice when I added the commit explicitly bringing it back so the commit explains why, but it does look odd in the diff so I can put it back to how it was at the start.